exercises/grains/src/example.h

```diff
@@ -0,0 +1,7 @@
+#ifndef _GRAINS_H
+#define _GRAINS_H
+
+unsigned long square(int sq);
+unsigned long total(void);
```

Comment:
I personally would prefer using fixed-width types, but all of the other exercises seem to favor  primitive types.  What's the consensus?  Should we pull in `stdint.h` and use `uint64_t`?